helm/rook-operator/values.schema.json

```diff
@@ -53,7 +53,21 @@
                     "type": "boolean"
                 },
                 "pluginTolerations": {
-                    "type": "array"
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "effect": {
+                                "type": "string"
+                            },
+                            "operator": {
+                                "type": "string"
+                            }
+                        }
+                    }
+                },
```

Comment:
The upstream chart doesn't set any plugin tolerations, but we need to ignore `NoSchedule` taints in GS clusters.